src/components/LandingPage/sections/SectionHome/WholeHistory/PortalsPanel/PortalsPanel.tsx

```diff
@@ -0,0 +1,105 @@
+  className?: string;
+}
+
+export default function PortalsPanel({ className }: PortalsPanelProps) {
+  return (
+    <>
+      <h1>Browse through the portals built by the Blue Brain Project</h1>
+      <CenteredColumn>
+        <div className={classNames(className, styles.portalsPanel)}>
+          {PORTALS.map(({ title, content, image, href }) => (
+            <Link key={title} href={href} className={styles.center}>
+              <div className={styles.button}>
+                <div className={styles.text}>
+                  <div>Portal</div>
+                  <h2>{title}</h2>
+                  <div>{content}</div>
+                </div>
+                <div className={styles.image} style={{ backgroundImage: `url(${image})` }} />
+              </div>
+            </Link>
+          ))}
+        </div>
+      </CenteredColumn>
+    </>
+  );
+}
+
+const PORTALS: Array<{
+  title: string;
+  content: string;
+  href: string;
+  image: string;
+}> = [
+  {
+    title: 'The Neocortical Microcircuit Collaboration Portal',
+    content:
+      "An online public resource of the Blue Brain Project's first release of a digital reconstruction of the microcircuitry of juvenile Rat somatosensory cortex",
+    image: NeocorticalURL.src,
+    href: 'https://bbp.epfl.ch/nmc-portal/welcome.html',
+  },
+  {
+    title: 'The Neuro-Glia-Vasculature Portal',
+    content:
+      'Come and explore the various datasets and visuals made available to understand how we reconstruct in silico the Neuro-Glia-Vasculature ensemble architecture of the rat brain.',
+    image: NgvURL.src,
+    href: 'https://bbp.epfl.ch/ngv-portal/',
+  },
+  {
+    title: 'The hippocampus hub',
+    content: 'Explore the existing Hippocampus model and all its parts!',
+    image: HippocampusURL.src,
+    href: 'https://www.hippocampushub.eu/model/',
+  },
+  {
+    title: 'The Blue Brain Cell Atlas',
+    content:
+      'The Blue Brain Cell Atlas is a comprehensive online resource that describes the number, types, and positions of cells in all areas of the mouse brain.',
+    image: AtlasURL.src,
+    href: 'https://bbp.epfl.ch/nexus/cell-atlas/',
+  },
+  {
+    title: 'Channelpedia',
+    content:
+      'Web-based freely-accessible information management network and electrophysiology data repository for comprehensive ion channel research.',
+    image: ChannelURL.src,
+    href: 'https://channelpedia.epfl.ch',
+  },
+  {
+    title: 'The SSCx portal',
+    content: "Discover a tissue-level model of the rodent somatosensory cortex 'in silico'",
+    image: SscxURL.src,
+    href: 'https://bbp.epfl.ch/sscx-portal/',
+  },
+  {
+    title: 'The Thalamus studio',
+    content:
+      'Thalamic control of sensory processing and spindles in a biophysical somatosensory thalamoreticular circuit model of wakefulness and sleep.',
+    image: ThalamusURL.src,
+    href: 'https://bbp.epfl.ch/nexus/web/studios/public/thalamus/studios/e9ceee28-b2c2-4c4d-bff9-d16f43c3eb0f',
```

Comment:
This link doesn't work.

The studio itself is deployed under https://bbp.epfl.ch/thalamus-studio, there supposed to be a redirect from the old link (the one you've used and that was probably published somewhere in papers/articles) to the simplified one that I've posted.

I know Jean-Denis was asking about the redirect couple of days ago, maybe EPFL will add one, but for the platform, maybe we should link directly to the current deployment?

Reply:
Good point. We can use this link.